src/main/java/picard/analysis/directed/PanelMetricsBase.java

```diff
@@ -151,15 +151,15 @@ public class PanelMetricsBase extends MultilevelMetrics {
     /**
      * A measure of how undercovered <= 50% GC regions are relative to the mean. For each GC bin [0..50]
      * we calculate a = % of target territory, and b = % of aligned reads aligned to these targets.
-     * AT DROPOUT is then abs(sum(a-b when a-b < 0)). E.g. if the value is 5% this implies that 5% of total
+     * AT_DROPOUT is then sum(abs(b-a)) when b-a < 0. E.g. if the value is 5%, this implies that 5% of total
```

Comment:
I think it would be clearer to write this as `AT_DROPOUT is then sum(a-b if a-b > 0 else 0)`.

Reply:
Done